### ISSUE TITLE
Tracking with LAP crashes with IndexError

### ISSUE THREAD

**Issue by emmenlau** [CellProfiler/CellProfiler#2103]
I'm tracking cells from a time series multi-page TIFF file, in LAP tracking mode, default parameters. In post-processing, it crashes with the error message:

```
Traceback (most recent call last):
  File "/home/emmenlau/BioDataAnalysis/Projects/TargetInfectX/CellProfiler/GIT-CellProfiler2/cellprofiler/pipeline.py", line 2359, in post_group
    module.post_group(workspace, grouping)
  File "/home/emmenlau/BioDataAnalysis/Projects/TargetInfectX/CellProfiler/GIT-CellProfiler2/cellprofiler/modules/trackobjects.py", line 1459, in post_group
    self.recalculate_kalman_filters(workspace, image_numbers)
  File "/home/emmenlau/BioDataAnalysis/Projects/TargetInfectX/CellProfiler/GIT-CellProfiler2/cellprofiler/modules/trackobjects.py", line 2886, in recalculate_kalman_filters
    next_object_number[next_has_link]
IndexError: arrays used as indices must be of integer (or boolean) type
```

**Comment by braymp** [comment]
I've come across this myself; definitely a show-stopper. I'm attaching some images and a pipeline to replicate the error.
[objects+pipelines.zip](https://github.com/CellProfiler/CellProfiler/files/797208/objects.pipelines.zip)

**Comment by bethac07** [comment]
@emmenlau @braymp Did you ever solve this?  Just ran into it with @cells2numbers too. :-/

**Comment by emmenlau** [comment]
Actually I did not test anymore, so its very well possible that the issue persits :-(

**Comment by braymp** [comment]
I'm sure the problem is still there; I def. leaned on Lee for diagnosing TrackObj problems.

**Comment by DeboraOlivier** [comment]
![image](https://user-images.githubusercontent.com/52743133/92362879-f2876c00-f0f0-11ea-8c96-a288e47a4ce1.png)
 I get this error too and not sure what it is due to.
I run
- CP 3.1.9
- LAP tracker in addition to various segmentations
- 17 workers (out of 20) 
- I get the error at i,qge 417 out of 418
- then the program gets stuck saving it seems ("Post-processing, please wait" is stuck) - had to cancel the analysis (and that got stuck too and had to "end task"



Tracking overall seems fine as the pipelines generates tracked images but just not the csv...

![image](https://user-images.githubusercontent.com/52743133/92363221-793c4900-f0f1-11ea-94d9-e1661e453776.png)

**Comment by braymp** [comment]
An earlier commit (https://github.com/braymp/CellProfiler/commit/5a8ed3b0985a5c34f970103cf164faef3a76d2a4) should fix this, and I believe it will be included in the v4.0 release.

**Comment by DeboraOlivier** [comment]
might wait for 4.0, saw it might come out very soon :-)
Also, stupid question, how can I include that commit into my CP? I downloaded the stable version here, not source install - would I need to source install and specify this branch?
